### DISCUSSION TITLE
Android demo 缺失图片右侧文字

### DISCUSSION THREAD

**Discussion by runboktu** [PaddlePaddle/PaddleOCR#12814] [Q&A]
使用Release/2.7分支的Android demo，代码模型未做修改，识别问题如下：
识别图片：
<img width="285" alt="屏幕快照 2023-10-10 下午4 17 26" src="https://github.com/PaddlePaddle/PaddleOCR/assets/22660918/8322421b-b160-4b8e-b451-e277447dc7b8">

部分识别结果如下(图片右侧文字都缺失)：
Det: (50,1603) (1026,1603) (1026,1652) (50,1652) 
 Rec: 黎巴嫩真主党向以色列发,0.9880953 Cls: 0,0.99999523
Det: (48,1443) (182,1446) (180,1505) (47,1502) 
 Rec: 人员。,0.9937522 Cls: 0,1.0
Det: (52,1363) (1022,1365) (1022,1414) (52,1412) 
 Rec: 以色列，以色列军方在边,0.9983041 Cls: 0,0.9999995
Det: (54,1276) (1024,1275) (1024,1324) (54,1325) 
 Rec: 息，有武装人员企图穿越,0.99792194 Cls: 0,0.9999999
Det: (54,1191) (1024,1193) (1024,1239) (54,1237) 
 Rec: 当天稍早时，以色列军方,0.9717401 Cls: 0,1.0
Det: (54,1038) (330,1038) (330,1083) (54,1083) 
 Rec: 取任何行动。,0.9735616 Cls: 0,1.0
Det: (52,951) (1024,951) (1024,1000) (52,1000) 
 Rec: 主党官员则表示，该组织,0.9959949 Cls: 0,1.0
Det: (54,864) (1026,864) (1026,913) (54,913) 
 Rec: 生交火，数名武装人员被,0.99394375 Cls: 0,1.0

**Comment by pinguoooo** [comment]
遇到了同样的问题，请问解决了吗？
